charts/zeppelin/templates/server/configmap.yaml

```diff
@@ -0,0 +1,72 @@
+{{- /*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/ -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "zeppelin.server.configMapName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "zeppelin.server.labels" . | nindent 4 }}
+data:
+  zeppelin-env.sh: |
+    #!/bin/bash
+    #
+    # Licensed to the Apache Software Foundation (ASF) under one or more
+    # contributor license agreements.  See the NOTICE file distributed with
+    # this work for additional information regarding copyright ownership.
+    # The ASF licenses this file to You under the Apache License, Version 2.0
+    # (the "License"); you may not use this file except in compliance with
+    # the License.  You may obtain a copy of the License at
+    #
+    #    http://www.apache.org/licenses/LICENSE-2.0
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
+    #
+
+    export SPARK_MASTER=k8s://https://kubernetes.default.svc
+    export ZEPPELIN_K8S_SERVICE_NAME={{ include "zeppelin.server.serviceName" . }}
+    export ZEPPELIN_K8S_SPARK_CONTAINER_IMAGE={{ include "zeppelin.interpreter.spark.image" . }}
+
+  zeppelin-site.xml: |
```

Comment:
Please carry out a configuration on environment variables. This configuration is easier to understand and does not require an additional xsl file.

Reply:
I think it would be better to provide both methods. One can choose to configure the Zeppelin conf with `server.conf` or configure the environment variables with `server.env` / `server.envFrom`  value, and let users to choose whatever they like.